ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/private-hosted-zone/deprovision/cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-deprovision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-private-hosted-zone-deprovision
+  steps:
+  - ref: gather-aws-console
+  - chain: gather
+  - ref: ipi-deprovision-deprovision
+  - ref: aws-deprovision-stacks
+  - ref: aws-deprovision-route53-private-hosted-zone
```

Comment:
since the private zone is being associated with VPC, is it possible to remove VPC before de-associate/remove private zone?
Per my understanding, the resources should be removed in reverse order of creation

Reply:
I have checked manually, It is possible to remove VPC before de-associate/remove private zone.